Bottom: c78cb77185acea915558d8e2be4a0c660cc4cf6f
Top:    8e802d4f707fce7ac97b131592181ea26a66a664
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-07 14:12:15 -0500



---
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
